tests/functionTests.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require ('../functions.php');
```

Comment:
this file includes your db connection, which means you need the database to run the tests. this is bad practise